modules/exploits/multi/http/atutor_sqli.rb

```diff
@@ -243,7 +243,7 @@ def get_ascii_value(sql, cookie)
        mid = (lower + upper) / 2
        sqli = "#{sql}>#{mid}"
        result = perform_request(sqli, cookie)
-       if result =~ /There are \d entries./
+       if result =~ /There are \d{0,8} entries\./
```

Comment:
I think he means ```\d{0,8}```. Because ```[0-8]``` would mean you can only have no more than 9 students, right?